chi/lease.py

```diff
@@ -598,7 +598,11 @@ def add_network_reservation(
             "network_name": network_name,
             "network_description": ",".join(desc_parts),
             "resource_properties": json.dumps(
-                ["==", "$physical_network", physical_network]
+                [
+                    "and",
+                    ["==", "$physical_network", physical_network],
+                    ["==", "$usage", "none"]
```

Comment:
The "correct" defaults to set here will actually vary per site, as "usage=none" will fail when `usage` isn't set on any networks.

This default should be set in the site `defaults.yml`, but I don't think we can set it here.